build

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
Can you give some reasoning for this change?

Reply:
This is needed for the `source` command that only exists for bash. Would you prefer to keep `sh` and use `.` instead of `source`?

Reply:
Yes please, `.` is POSIX.